<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QMainWindow" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1083</width>
    <height>2244</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow
{
	
	background-color: rgb(241, 241, 241);
}
#msgEdit
{
	
	border-image: url(:/sendmsg.png);
}
#pushButton
{
	border-image: url(:/sBt.png);

}
#pushButton:press
{
	border-image: url(:/sBt_hover.png);

}
#voiceBt
{
	
	border-image: url(:/voice.png);
}
#voiceBt:press
{
	
	border-image: url(:/voice2.png);
}
#imagBt
{
	
	border-image: url(:/photo.png);
}
#imagBt:press
{
	
	border-image: url(:/photo.png);
}
#photoBt
{
	
	border-image: url(:/cam.png);
}

#photoBt:press
{
	
	border-image: url(:/cam2.png);
}
#fileBt
{
	
	border-image: url(:/file.png);
}
#fileBt:press
{
	
	border-image: url(:/file2.png);
}


#DesktopBt
{
	
	border-image: url(:/Rcontrl .png);
}

#DesktopBt:press
{
	
	border-image: url(:/Rcontrl_hover .png);
}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>20</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="ipEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>60</horstretch>
            <verstretch>40</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>450</width>
            <height>48</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>100</height>
           </size>
          </property>
          <property name="text">
           <string>192.168.43.93</string>
          </property>
          <property name="placeholderText">
           <string>ip</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="portEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>450</width>
            <height>48</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>100</height>
           </size>
          </property>
          <property name="text">
           <string>8080</string>
          </property>
          <property name="placeholderText">
           <string>port</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="DesktopBt">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>64</width>
            <height>64</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>64</width>
            <height>64</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QTextBrowser" name="rcvBrowser">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(241, 241, 241);</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_9">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QLineEdit" name="msgEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>720</width>
            <height>128</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>720</width>
            <height>128</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>30</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>317</width>
            <height>128</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>315</width>
            <height>128</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Palatino Linotype</family>
            <pointsize>18</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::StrongFocus</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);
font: 18pt &quot;Palatino Linotype&quot;;</string>
          </property>
          <property name="text">
           <string>send</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer_10">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1083</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>back</string>
    </property>
    <addaction name="actionback"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="actionback">
   <property name="text">
    <string>back</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
